docs/pages/identity-security/usage/alerts.mdx

```diff
@@ -0,0 +1,139 @@
+---
+title: Identity Activity Center Alerts
+sidebar_label: Alerts
+description: Describes the pre-built security detections that trigger alerts in Teleport Identity Activity Center.
+tags:
+ - identity-security
+---
+
+Teleport Identity Security provides 50 pre-built security detections that automatically create alerts for suspicious identity-related activities across your infrastructure. These detections monitor events from Teleport and integrated services like AWS, GitHub, and Okta to identify potential security risks.
```

Comment:
Do we need to call out the specific number of detections? I don't think knowing the exact number adds any value, and it is likely to go out of date as we refine them in the future.